app/components/common/navbar/big_navbar/NavBarBig.js

```diff
@@ -3,14 +3,10 @@ import Box from "@mui/material/Box";
 import Drawer from "@mui/material/Drawer";
 import Button from "@mui/material/Button";
 import List from "@mui/material/List";
-import ListItem from "@mui/material/ListItem";
-import ListItemButton from "@mui/material/ListItemButton";
-import ListItemIcon from "@mui/material/ListItemIcon";
-import ListItemText from "@mui/material/ListItemText";
+
 import SmallFooter from "../SmallFooter";
 import NAVBAR_LINKS from "@/components/lists/navbar_links_map";
 import Link from "next/link";
-import Image from "next/image";
 
 export default function NavBarBig() {
```

Comment:
is nav bar big really the best name for this component?

Reply:
No, this name is not the most appropriate (not clear). Thank you! Correction done